examples/data-visualization/index.ts

```diff
@@ -0,0 +1,165 @@
+import { Viewer, NavigationCube, Heatmap, ContinuousHeatmapChannel, HeatmapSource, Icons, CameraType, ViewType, IHeatmapChannel, ClippingPlane, ProductType, } from '../..';
+import { Icon } from '../../src/plugins/DataVisualization/Icons/icon';
+import { Grid } from '../../src/plugins/Grid/grid';
+import { IconsData } from './icons';
+
+const viewer = new Viewer("viewer");
+const cube = new NavigationCube();
+const grid = new Grid();
+const heatmap = new Heatmap();
+const icons = new Icons();
+cube.ratio = 0.05;
+cube.stopped = false;
+cube.passiveAlpha = 1.0;
+cube.trueNorth = 0;
+viewer.addPlugin(grid);
+viewer.addPlugin(cube);
+viewer.addPlugin(heatmap);
+viewer.addPlugin(icons);
+
+const tempChannelId = "room_temp";
+const humidityChannelId = "room_humidity";
+
+const temperatureSource = new HeatmapSource("Temp sensor", 1, 152, tempChannelId, 1);
+const humiditySource = new HeatmapSource("Humidity sensor", 1, 152, humidityChannelId, 10);
+const sourceIcon = new Icon("Room 1 Sensor", "Temprature sensor", 1, 152, IconsData.errorIcon);
+
+let selectedChannel: ContinuousHeatmapChannel;
+const tempChannel = new ContinuousHeatmapChannel
+(tempChannelId, "double", "Temprature", "Temperature of Rooms", "temprature", "°F", 0, 150, ["#142ce2","#fff200","#FF0000"]);
```

Comment:
Let's make this °C and use 0-40 as the range (and update the Random generator below)